### PR TITLE
detect simulator start time using system log

### PR DESCRIPTION
**Pull request by Jonahss**
@imurchie for review.

### CODE REVIEW

File: lib/simulator-xcode-6.js [hunk 1]
```diff
@@ -178,25 +180,72 @@ class SimulatorXcode6 {
     return _.compact(existence).length !== files.length;
   }
 
+  async getBootedIndicatorString () {
+    let indicator = '';
```
Comment (imurchie) [inline lib/simulator-xcode-6.js]: This can just be a declaration, since there is no case that will not be assigned in the switch statement.

**Comment by imurchie** [comment]
One comment, which doesn't need to be acted upon. Otherwise, looks good!

File: lib/simulator-xcode-6.js (hunk 1 above)
Comment (Jonahss) [inline lib/simulator-xcode-6.js]: :+1:

**Comment by Jonahss** [comment]
@imurchie fixed this PR. using a better method to tail things instead of tailing-stream

**Comment by imurchie** [comment]
Interesting. Pretty cool.

**Comment by Jonahss** [comment]
@imurchie hopefully thats the last change.

File: lib/tail-until.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { SubProcess } from 'teen_process';
+import B from 'bluebird';
+
+// tails a file, promise resolves when input string is written to file
+async function tailUntil(filePath, until, timeout = 5000) {
```
Comment (imurchie) [inline lib/tail-until.js]: Space before arguments.

**Comment by Jonahss** [comment]
@imurchie how's it look now?

**Comment by imurchie** [comment]
Looks good!

**Pull request merged**